### PR TITLE
changed license assignment instructions

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/427

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: Teams/teams-administration/failed-to-auto-discover-domain-error-teams-admin-center.md [hunk 1]
```diff
@@ -45,16 +45,14 @@ This fix can take from 15 minutes to an hour to take effect.
...
 
-1. Sign in to https://portal.office.com.
-2. Navigate to **Admin** > **Users**.
-3. Select **Active users** > **Product license**.
-4. Select **Edit**, and then select **Turn on Skype for Business license**.
-
-If there is an Office 365 E5 plan, expand the available products list, and then select **Skype for Business license**. The license must be active together with Microsoft Teams.​​
+1. Sign in to https://admin.microsoft.com.
+2. Navigate to **Users** > **Active Users**.
+3. Select admin account you want to assign license to and press **Manage product licenses**.
```
Comment (JohanFreelancer9) [inline Teams/teams-administration/failed-to-auto-discover-domain-error-teams-admin-center.md:54]:
```suggestion
3. Select the admin account you want to assign a license to and then select **Manage product licenses**.
```

File: Teams/teams-administration/failed-to-auto-discover-domain-error-teams-admin-center.md [hunk 2]
```diff
@@ -45,16 +45,14 @@ This fix can take from 15 minutes to an hour to take effect.
...
+4. In the apps list, enable **Skype for Business Online** or **Microsoft Teams** checkboxes and press **Save Changes**.
```
Comment (JohanFreelancer9) [inline Teams/teams-administration/failed-to-auto-discover-domain-error-teams-admin-center.md:55]:
```suggestion
4. In the apps list, check either the **Skype for Business Online** or **Microsoft Teams** checkboxes and select **Save Changes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**